src/components/Filter.js

```diff
@@ -285,7 +285,7 @@ function Filter(props) {
             value={dateStart}
             onChange={handleDateStartChange}
             format={getDateFormatLocale(true)}
-            maxDate={dateEnd}
+            maxDate={dateEnd || Date('2100-01-01')}
```

Comment:
Perhaps we could add these default values to the state on line 58-59 so we can keep the initial state and it's defaults in one place?

Reply:
If I've understood your suggestion correctly, that doesn't have quite the desired result.
We want to keep the actual `dateStart` and `dateEnd` unset so that no value is selected in the input.
It's only the `minDate` and `maxDate` props that we need to set.

Reply:
Ah, I misunderstood!  Ok, that solution works.  What do you think about adding comments to explain why the dates are there? or to create variables maxDateDefault and minDateDefault?  I'm concerned with someone changing or deleting the defaults in a refactor.
This would be a good thing to have a test for...

Reply:
That's a good shout, thanks. I'll make it a bit more explicit and add a test or two.